ledger/src/versions/common/mod.rs

```diff
@@ -468,7 +475,31 @@ where
 		tx: &[u8],
 		block_context: &BlockContext,
 	) -> Result<(StorageCost, GasCost), LedgerApiError> {
```

Comment:
Probably good to change the return type to a single value here - the ledger cost combines storage and compute cost